src/NuGetPackageVerifier/Rules/FrameworkAssembliesDoesNotContainFacadesRule.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using NuGet.Frameworks;
+using NuGet.Packaging;
+using NuGetPackageVerifier.Logging;
+
+namespace NuGetPackageVerifier.Rules
+{
+    public class FrameworkAssembliesDoesNotContainFacadesRule : IPackageVerifierRule
+    {
```

Comment:
:smile: @Eilon was concerned about speed and flakiness, so I kept it super simple.